core/vtk/ttkCompactTriangulationPreconditioning/ttkCompactTriangulationPreconditioning.cpp

```diff
@@ -0,0 +1,207 @@
+#include <ttkCompactTriangulationPreconditioning.h>
+
+#include <vtkCellData.h>
+#include <vtkDataArray.h>
+#include <vtkDataSet.h>
+#include <vtkInformation.h>
+#include <vtkInformationVector.h>
+#include <vtkPointData.h>
+#include <vtkSmartPointer.h>
+
+#include <ttkMacros.h>
+#include <ttkUtils.h>
+
+// A VTK macro that enables the instantiation of this class via ::New()
+// You do not have to modify this
+vtkStandardNewMacro(ttkCompactTriangulationPreconditioning);
+
+ttkCompactTriangulationPreconditioning::
+  ttkCompactTriangulationPreconditioning() {
+  this->setDebugMsgPrefix("ttkCompactTriangulationPreconditioning");
```

Comment:
please remove this line.
differentiating the base and vtk layers in the messages has been deprecated since the second version of the internal log API.

Reply:
solved.